R/fit.R

```diff
@@ -911,3 +910,64 @@ CmdStanVB <- R6::R6Class(
   )
 )
 
+CmdStanGQ <- R6::R6Class(
+  classname = "CmdStanGQ",
+  inherit = CmdStanFit,
+  public = list(
+    generated_quantities = function(variables = NULL) {
```

Comment:
What should draws return for CmdstanGQ? The input fitted_params? Or should draws() just return the generated_quantities in which case this is not needed?

Reply:
> Or should draws() just return the generated_quantities in which case this is not needed?

This, I think. I don't think we need to introduce a new method since the name "draws" still makes sense when referring to generated quantities (they're still draws from a distribution). So I would say just use `$draws()` for this.

Reply:
Ok, thanks. Sounds reasonable.

cmdstanpy returns generated quantities + the input fitted params. If there is a name conflict the generated quantity is used. I am not a huge fan of that but I can also see it being useful. Though one can always combine the fitted params and generated quantities on their own with posterio (bind by iteration). We could show that example in a vignetter. What do you think?

Reply:
> I am not a huge fan of that

I agree with you, it should just return the generated quantities. Yeah let's just show an example of combining them together in the doc.